src/pages/contributors/ContributorsTable.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import './ContributorsTable.css';
+
+export function ContributorsTable(){
+    return (
+        <table>
+            <tr>
+                <td>#</td>
+                <td>Username</td>
+                <td>GitHub</td>
+            </tr>
+            <tr>
+                <td>1</td>
+                <td>pavelkeyzik</td>
+                <td>https://github.com/pavelkeyzik</td>
```

Comment:
It must be a link element. Please, use `<a href="...">` element

Reply:
Whether I have to make the whole row a link or just Github column element.

Reply:
Just GitHub column element